app/controllers/api/v1/accounts/conversations/assignments_controller.rb

```diff
@@ -14,7 +14,8 @@ def create
 
   def set_agent
     @agent = Current.account.users.find_by(id: params[:assignee_id])
-    @conversation.update_assignee(@agent)
+    @conversation.assignee = @agent
+    @conversation.save!
```

Comment:
Updating this just because I removed the update_assignee method in conversation.